orte/mca/rml/base/rml_base_frame.c

```diff
@@ -213,7 +218,7 @@ int orte_rml_base_select(void)
             opal_output(0, "\tComponent: %s Priority: %d", mod->component->mca_component_name, mod->pri);
         }
    }
-
+  
```

Comment:
Another nit-pick: Please don't add lines with whitespace at the end.  We've actually done entire passes through the code base to remove all whitespace like this.  Thanks.